renderdoc/driver/vulkan/vk_pixelhistory.cpp

```diff
@@ -3329,7 +3338,13 @@ struct VulkanPixelHistoryPerFragmentCallback : VulkanPixelHistoryCallback
         m_pDriver->ReplayDraw(cmd, *action);
         state.EndRenderPass(cmd);
 
-        if(i == 1)
+        // Restore location mapping for the other pipelines.
+        if(isPrimitiveIDPipe && prevState.dynamicRendering.localRead.AreLocationsNonDefault())
```

Comment:
The reason I did this is because DRLR may not always be available. If the values are non-default, it means DRLR was definitely used and the extension was available.

Would you prefer I replace this with an extension check?